lib/camp.js

```diff
@@ -712,6 +712,10 @@ function Path(matcher) {
       return path.match(matcher);
     };
   } else if (typeof matcher === 'string') {
+    if (matcher[0] !== '/') {
+      matcher = '/' + matcher;
+      this.matcher = matcher;
```

Comment:
The idea is to have `this.matcher` be as close to the source material as possible, to allow introspection (which handler produces this?) and planned performance improvements.